uwsgi/run.sh

```diff
@@ -21,7 +21,14 @@ elif [ $1 == "cleanup-db" ]; then
         echo "\${MAX_AGE} must be set!"
         exit 1
     fi
-    exec python scripts/manage-db.py -d ${DBURI} cleanup ${MAX_AGE}
+    if [ -z "${DELETE_RUN_TIME}" ]; then
+        echo "\${DELETE_RUN_TIME} must be set!"
+        exit 1
+    fi
+
+    exec scripts/run-batch-deletes.sh $DBURI $MAX_AGE $DELETE_RUN_TIME
+
+    #exec python scripts/manage-db.py -d ${DBURI} cleanup ${MAX_AGE}
 elif [ $1 == "cleanup-db-dryrun" ]; then
```

Comment:
I changed the design enough that the dryrun is likely to confuse. Maybe renaming it to smoketest? Are we bikeshedding?